lib/Dialect/Torch/Transforms/ShapeLibrary.cpp

```diff
@@ -29,11 +29,11 @@ StringRef mlir::torch::Torch::getShapeLibrary() {
   constexpr StringLiteral shapeLib(R"mlir(
 module {
   func.func @"__torch_mlir_shape_fn.aten.tanh"(%arg0: !torch.list<int>) -> !torch.list<int> {
-    %0 = func.call @__torch__.torch_mlir.dialects.torch.importer.jit_ir.build_tools.upstream_shape_helpers.unary(%arg0) : (!torch.list<int>) -> !torch.list<int>
```

Comment:
Do you know what happened here?

Reply:
hmm, I don't know it seems like all func.call got changed to call